paychmgr/simple.go

```diff
@@ -283,25 +283,16 @@ func (ca *channelAccessor) msgWaitComplete(mcid cid.Cid, err error) {
 	// The queue may have been waiting for msg completion to proceed, so
 	// process the next queue item
 	if len(ca.fundsReqQueue) > 0 {
-		go ca.processQueue() // nolint: errcheck
+		go ca.processQueue("") // nolint: errcheck
 	}
 }
 
-func (ca *channelAccessor) currentAvailableFunds(queuedAmt types.BigInt) (*api.ChannelAvailableFunds, error) {
```

Comment:
Moved the logic for what to do if the channel is not found into the manager layer